### PR TITLE
TVR-23910 Concur Request doc update

### PR DESCRIPTION
**Pull request by laurentlhermine**
Introduce isPaymentCardRelated field in v4 APIs and events
Update employee schema doc to remove href and template

### CODE REVIEW

File: src/api-explorer/v4-0/ConcurRequest.swagger2.json [hunk 1]
```diff
@@ -3575,6 +3575,10 @@
           "type": "string",
           "description": "The unique identifier of the expected expense"
         },
+        "isPaymentCardRelated": {
+          "type": "boolean",
+          "description": "Indicates if the expected expense entry has the `Is Payment Card Related` field enabled"
```
Comment (marcdalleau) [inline src/api-explorer/v4-0/ConcurRequest.swagger2.json:3580]: I'm seeing two different quotation marks here `Is Payment Card Related` and ‘sendback’ 
are you sure the text is displayed as expected

File: src/api-reference/request/v4.endpoints.schemas.markdown [hunk 2]
```diff
@@ -260,6 +258,7 @@ Name|Type|Format|Description
 `expenseType`|`object`|[`Expense Type`](#schema-expensetype)|The expense type of the entry. **Required** for expected expenses, automatically set for segments depending on the `SegmentType` code.
 `href`|`string`|[RFC 3986]|Hyperlink to the resource for this Request entry.
 `id`|`string`|-|The unique identifier of the expected expense entry.
+`isPaymentCardRelated`|`boolean`|-|Indicates if the expected expense entry has the `Is Payment Card Related` field enabled. Value is `true` when the end user has selected this field for this expected expense entry.
```
Comment (marcdalleau) [inline src/api-reference/request/v4.endpoints.schemas.markdown:261]: seems to be similar everywhere except for [ConcurRequest.swagger2.json](https://github.com/SAP-docs/preview.developer.concur.com/pull/1100/files#diff-6717e90089037c3a139c2e3e0fafed032a3cc61d11d14105f75d9929122fec7d) line 4796

File: src/api-explorer/v4-0/ConcurRequest.swagger2.json (hunk 1 above)
Comment (laurentlhermine) [reply src/api-explorer/v4-0/ConcurRequest.swagger2.json]: fixed

**Review by marcdalleau — APPROVED** [review]

**Comment by RoAnnC1** [comment]
@laurentlhermine Since the last time you updated the swagger we have implemented a new process that runs a bunch of automated tests that must pass before this can be published on API Hub. Please submit your swagger here: https://github.concur.com/developer/swagger.

**Comment by laurentlhermine** [comment]
> @laurentlhermine Since the last time you updated the swagger we have implemented a new process that runs a bunch of automated tests that must pass before this can be published on API Hub. Please submit your swagger here: https://github.concur.com/developer/swagger.

Yes please check https://github.concur.com/developer/swagger/pull/43
Since the existing files had reformatted the original swagger JSON with a mix of 4 spaces tab and 2 spaces tab I've decided to reformat the whole files as RFC 8259 and a 2 spaces tab standard. I know it makes the review in Github a bit hard (need to ignore whitespaces). Hope this works for you.

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**